src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLAlertImpl.java

```diff
@@ -156,7 +156,7 @@ public DataFetcher<Iterable<Object>> entities() {
         .map(entitySelector -> {
           if (entitySelector instanceof EntitySelector.Stop) {
             FeedScopedId id = ((EntitySelector.Stop) entitySelector).stopId;
-            StopLocation stop = getTransitService(environment).getStopForId(id);
+            StopLocation stop = getTransitService(environment).getRegularStop(id);
```

Comment:
I don't really have a need currently to support flex stops here (maybe @leonardehrenfried does?) but theoretically maybe it should be possible to target an alert to a flex location also. Was it that the getStopForId only returned "regular stops"? If it this doesn't change the behaviour, I don't have a problem with this as of now. There are other instances in the legacy GraphQL code where a stop is only fetched with the `getRegularStop` method which if I understand correctly doesn't support flex stops.

Reply:
I think it is best to be future complient if if comes with a very small overhead like this one (very fast to check the to extra maps). So I will change it.